www/data/src/readme.md

```diff
@@ -8,7 +8,7 @@ james_tompkin@brown.edu - Any issues, email me.
 This software takes the ETRA2018 dataset and creates CSV files for each video, containing per-frame WebGazer and Tobii values in normalized screen coordinates.
 After extraction, this makes it simple and efficent to analyse the performance of WebGazer in your favourite _data science_ application.
 
-Works with webgazer.js commit 01a9c68cf0a4b1f3a6d6910bf86d1ff403831ec7 (and onwards) from 30th July 2018 11:49 PM.
+Works with webgazer.js commit 5a12c64db54b489cda9bad5a1b90fa9ccfdde4e8
```

Comment:
Grr... this commit ID will be changed once it's squashed, this ReadMe change will likely need to be a second PR.